buildkite/bazelci.py

```diff
@@ -81,6 +81,8 @@
     "bazel": "gs://bazel-kzips/",
 }[BUILDKITE_ORG]
 
+BAZELCI_ENABLE_REMOTE = os.environ.get("BAZELCI_ENABLE_REMOTE", "true") == "true"
```

Comment:
Please add a comment to explain what this env var means. Is it better to rename it to something like `BAZELCI_LOCAL_RUN`?

Reply:
Done.